go.mod

```diff
@@ -6,49 +6,48 @@ require (
 	github.com/container-storage-interface/spec v1.5.0
 	github.com/golang/protobuf v1.5.2
 	github.com/kubernetes-csi/csi-lib-utils v0.9.0
-	github.com/kubernetes-csi/external-snapshotter/v2 v2.0.0-20200617021606-4800ca72d403
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
 	github.com/pborman/uuid v1.2.0
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/net v0.0.0-20210825183410-e898025ed96a
+	golang.org/x/net v0.0.0-20211209124913-491a49abca63
 	google.golang.org/grpc v1.40.0
-	k8s.io/api v0.23.0
-	k8s.io/apimachinery v0.23.0
-	k8s.io/client-go v0.23.0
+	k8s.io/api v0.23.3
```

Comment:
We normally update the dependencies with every K8s release.  This was updated to 0.23.0 already.  Is there anything specific in 0.23.3 that we need right now or can we wait until 1.24 release to update?

Reply:
ok, I see, let me only update mount-utils package change(with cherry-pick fix) in v1.23.3

Reply:
If I only upgrade `mount-utils` to `v0.23.3`, then there is go mod error, so let's upgrade to v1.23.3 for all related modules since there is no harm, agree? @xing-yang 
```
# go mod tidy
go: k8s.io/kubernetes@v1.23.3 requires
        k8s.io/mount-utils@v0.0.0: reading k8s.io/mount-utils/go.mod at revision v0.0.0: unknown revision v0.0.0
go: k8s.io/kubernetes@v1.23.3 requires
        k8s.io/mount-utils@v0.0.0: reading k8s.io/mount-utils/go.mod at revision v0.0.0: unknown revision v0.0.0
```

Reply:
Sure